.github/workflows/lint.yaml

```diff
@@ -0,0 +1,31 @@
+name: lint
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.8"
```

Comment:
nit: do we also want to run on 3.9? Maybe not necessary for this initial version, but something to keep in mind

Reply:
hm idk running it x times will be slow right. let me see whats a "good practice" here